_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,284 @@
+---
+layout:       element
+title:        Forms
+date:         2015-05-12 16:02:35
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- forms
+
+slug:         forms
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/edx-pl/sass/components/_forms.scss"
+
+description:  Various examples of different form fields and their usage and options.
+
+info: Forms allow users to interact with the interface, set preferences, and enter any necessary personal information. They also form the base of individual exercises within each class.
+---
+
+<h3 class="hd-4 example-set-hd">Forms</h3>
+<h4 class="hd-6">Text inputs</h4>
+<div class="example-set">
+    <fieldset class="fieldset">
+        <legend class="group-legend is-hidden">Personal Details</legend>
+        <div class="field-wrapper">
```

Comment:
This class seems cumbersome - 1) we don't name wrappers in this way, its usually ``.wrapper-*`` and 2) should this even be a wrapper, as I don't see a direct element you're wrapping (usually we have a ``.field`` element)? I'd recommend moving this into two pieces of situational markup:

1. For single fields, use ``<div class="field">``
2. For a group of fields, use:

```html
<div class="field-group">
    <div class="field">
   </div>

    <div class="field">
   </div>
</div>
```